frontend/src/App.jsx

```diff
@@ -1,40 +1,21 @@
-import Counter from "./components/Counter";
-import logo from "./assets/logo.svg";
-
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Navbar from "./components/navbar/Navbar";
+import Home from "./pages/Home";
+import Equipe from "./pages/Equipe";
+import Jeu from "./pages/Jeu";
+import Contact from "./pages/Contact";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>Hello Vite + React !</p>
-
-        <Counter />
-
-        <p>
-          Edit <code>App.jsx</code> and save to test HMR updates.
-        </p>
-        <p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-          {" | "}
-          <a
-            className="App-link"
-            href="https://vitejs.dev/guide/features.html"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Vite Docs
-          </a>
-        </p>
-      </header>
+    <div className="container">
```

Comment:
This is the old way to use React-router-dom, please remove it for now.

Reply:
ok I will modify

Reply:
I deleted the react-router-dom. You can double-check